.travis.yml

```diff
@@ -18,8 +19,6 @@ cache:
   directories:
     - $HOME/.cache/pip
 
-stage: Comprehensive tests
```

Comment:
This bothered me also, but I think this labels the tests after the initial test. There may be another way to do it, but when I looked at it, it seemed that only the last entry was performed after a stage, so putting this into the jobs would not run all the remaining jobs in parallel. There may be a better way to do it, and frankly I'm not clear on how `stages: ...` works. The [documentation](https://docs.travis-ci.com/user/build-stages/) isn't that helpful. Note that the labeling of the second stage has gone missing in the current tests.

Reply:
I pushed a change that adds `stage: Comprehensive tests` to the second test listed under `jobs/include:`.  According to the docs, if a stage is not given, it uses the previous stage.  So if I understand the docs correctly, this should run the single python 3.8 test first in the "Initial tests" stage, and if it passes, all the other tests should run in parallel in the "Comprehensive tests" stage.

Reply:
It appears to be working: https://travis-ci.org/numpy/numpy/builds/627512705?utm_source=github_status&utm_medium=notification

Reply:
I'm wondering if having the comprehensive stage defined outside the jobs section accounted for the bad behavior before.

Reply:
Probably.  According to the linter, that line had no meaning.

Reply:
yeap, sorry about this. It used to be to set the default stage, but apparently it has changed (I just noticed this as the jobs from this default stage disappeared for astropy very recently).